src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/StorageAccount/SyncStorageAccount.cs

```diff
@@ -61,9 +68,15 @@ public sealed class SyncStorageAccount : AdminCmdlet
         /// </summary>
         [Parameter(Mandatory = false, Position = 8)]
         public string StorageAccountApiVersion { get; set; }
-        
+
+        /// <summary>
+        /// Specifies the Microsoft.Resource.Admin apiVersion
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 9)]
```

Comment:
No more than 3 or 4 parameters should be positional.  This should include the required parameters and the most common optional parameters.  Other optional parameters should neo be positional